Packs/Base/ReleaseNotes/1_39_27.md

```diff
@@ -0,0 +1,14 @@
+
+#### Scripts
+
+##### GetIncidentsByQuery
+
+- Fixed an issue where changes in the ***CommonServerUserPython*** script would cause an error in ***GetIncidentsByQuery***.
```

Comment:
I'm not sure this is relevant for all users as most of them don't use the `CommonServerUserPython`. In this case, it's better just to say `Maintenance and stability enhancements`.